cowprotocol/cowswap
# Added keystone to wallets modal

# Summary

Add Keystone wallet to wallets modal
Idea is to use Metamask connector for now and just show a button with a Keystone logo

# To test
- open wallets modal
- there should be a Keystone wallet button
- make sure it works properly and when connected that Keystone button is highlighted
